src/Commands/ProjectCreateCommand.php

```diff
@@ -190,11 +190,22 @@ public function validateCreateProject(CommandData $commandData)
      * @param string $target Simple name of project to create.
      * @option org Organization for the new project (defaults to authenticated user)
      * @option team Pantheon team
+     * @option label The friendly name to use for the Pantheon site (defaults to the name of the Git site).
      * @option pantheon-site Name of Pantheon site to create (defaults to 'target' argument)
      * @option email email address to place in ssh-key
+     * @option test-site-name The name to use when installing the test site.
+     * @option admin-email The email address to use for the admin.
+     * @option admin-password The password to use for the admin when installing the test site.
```

Comment:
@aaronbauman a small nitpick here. Please add CMS so folks know this is the Drupal/WordPress admin password (I've seen people get confused and think these options are associated with their Pantheon account)

Reply:
Added "CMS"